fix: pin studio version, PromptBasedClassify behaving weirdly (#1212)
Aleph-Alpha/intelligence-layer-sdk: CHANGELOG.md

...
### Features
...

### Fixes

>  - Add warning to `PromptBasedClassify` and `PromptBasedClassifyWithDefinitions` about being cautious when using them with models families other than luminous

Reviewer: ```suggestion
 - Add warning to `PromptBasedClassify` and `PromptBasedClassifyWithDefinitions` to be cautious when using them with model families other than luminous
```
